app/assets/javascripts/views/story_view.js

```diff
@@ -579,6 +476,174 @@ module.exports = FormView.extend({
+          story_state_options.push([ I18n.t('story.state.' + option), option ])
+        });
+        $(div).append(this.makeFormControl({
+          name: "state",
+          label: true,
+          control: this.select("state", story_state_options, {
+            attrs: {
+              class: [],
+              disabled: this.isReadonly()
+            }
+          })
+        }));
+      })
+    );
+
+    this.$el.append(
+      this.makeFormControl(function(div) {
+        $(div).addClass('form-inline');
+        $(div).append(this.makeFormControl({
+          name: "requested_by_id",
+          label: true,
+          control: this.select("requested_by_id",
+            this.model.collection.project.users.forSelect(), {
+              blank: '---',
+              attrs: {
+                class: [],
+                disabled: this.isReadonly()
+              }
+          })
+        }));
+        $(div).append(this.makeFormControl({
+          name: "owned_by_id",
+          label: true,
+          control: this.select("owned_by_id",
+            this.model.collection.project.users.forSelect(), {
+              blank: '---',
+              attrs: {
+                class: [],
+                disabled: this.isReadonly()
+              }
+          })
+        }));
+      })
+    );
+
+    this.$el.append(
+      this.makeFormControl({
+        name: "labels",
+        label: true,
+        control: this.textField("labels"),
+        class: 'form-control',
+        disabled: this.isReadonly()
+      })
+    );
+
+    this.$el.append(
+      this.makeFormControl(this.makeDescription())
+    );
+
+    this.renderTasks();
+
+    this.$el.append(
+      this.makeFormControl(function(div) {
+        var random = (Math.floor(Math.random() * 10000) + 1);
+        var progress_element_id = "documents_progress_" + random;
+        var finished_element_id = "documents_finished_" + random;
+        var attachinary_container_id = "attachinary_container_" + random;
+
+        $(div).append(this.label('attachments', I18n.t('story.attachments')));
+        $(div).addClass('uploads');
+        if(!this.isReadonly()) {
+          $(div).append(this.fileField("documents", progress_element_id, finished_element_id, attachinary_container_id));
+          $(div).append("<div id='" + progress_element_id + "' class='attachinary_progress_bar'></div>");
+        }
+        $(div).append('<div id="' + attachinary_container_id + '"></div>');
+
+        // FIXME: refactor to a separated AttachmentView or similar
+        // must run the plugin after the element is available in the DOM, not before, hence, the setTimeout
+        clearTimeout(window.executeAttachinaryTimeout);
+        window.executeAttachinaryTimeout = setTimeout(executeAttachinary, 500);
+      })
+    );
+
+    this.initTags();
+
+    this.renderNotes();
+  },
+
+  makeStoryTypeSelect: function(div) {
+    var story_type_options = _.map(["feature", "chore", "bug", "release"], function(option) {
```

Comment:
In the other one your were using `.each` and in this one you're using `.map`. Why two different approaches to basically the same task? And here we can see some code "duplication" (since you're taking different approches), if you create the function I previous mentioned you could eliminate that too.